src/test/scala/pirate.example/GitExample.scala

```diff
@@ -1,12 +1,14 @@
 package pirate.example
 
 import pirate._, Pirate._
-import scalaz._, effect.IO
+import scalaz._, Scalaz._, effect.IO
+
+import java.io.File
 
 case class Git(
-  cwd: String,
-  conf: String, // FIX support a read like thing for x=y
-  exec: String,
+  cwd: Option[String],
```

Comment:
They could instead have defaults. The parser had neither an optional or default clause which is one of the reasons it was failing, I just picked one.